doc/content/devices/configuring-devices/multicast/index.md

```diff
@@ -100,3 +100,10 @@ Then, schedule the following message to the [Application Server MQTT server]({{<
   ]
 }
 ```
+
+## How to update MAC Settings of a Multicast device?
+
+Changing MAC settings of a Multicast device (virtual ABP device) requires performing the below steps:
+
+1. Change [MAC settings]({{< ref "/devices/configuring-devices/mac-settings" >}}).
+2. Reset the Multicast device at the Network Server end as explained [here](/devices/configuring-devices/mac-settings/#session-and-mac-state-reset) to apply above mentioned changes.
```

Comment:
```suggestion
2. Reset the Multicast device at the Network Server end as explained [here]({{< ref "/devices/configuring-devices/mac-settings#session-and-mac-state-reset" >}}) to apply above mentioned changes.
```

Reply:
do you usually run docs locally to check if these links resolve? Because I'm pretty sure this one wouldn't resolve correctly

Reply:
Yes, we always check the rendered content and links by running the docs locally. The above short notation `[here](/devices/configuring-devices/mac-settings/#session-and-mac-state-reset)` resolves without any issues.
![image](https://github.com/TheThingsIndustries/lorawan-stack-docs/assets/105837895/2648b3bd-45d4-4ba4-96d5-394c37111dac)

However, we updated the above change.